site/app/models/gradeable/Gradeable.php

```diff
@@ -784,23 +784,16 @@ private static function validateDateSet(array $date_properties, array $date_valu
      */
     private function getDateValidationSet(bool $grade_inquiry_modified = false) {
         if ($this->type === GradeableType::ELECTRONIC_FILE) {
-            if (!$this->isStudentSubmit()) {
-                if ($this->isTaGrading()) {
-                    $result = self::date_properties_elec_exam;
-                }
-                else {
-                    $result = self::date_properties_bare;
-                }
-            }
-            elseif ($this->isTaGrading()) {
+            // $this->isStudentSubmit() == true indicates student can make multiple submissions
```

Comment:
I don't think we need to include this comment here since we are not using isStudentSubmit() anymore, and the meaning of 'student submit' is already commented on at line 231

Reply:
Got it, removed the comment
you may run the tests.